boa_parser/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "boa_parser"
+description = "ECMAScript parser for the Boa JavaScript engine."
+keywords = ["javascript", "js", "syntax", "parser"]
+categories = ["parser-implementations", "compilers"]
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+boa_interner.workspace = true
+boa_macros.workspace = true
+boa_ast.workspace = true
+boa_profiler.workspace = true
+boa_unicode.workspace = true
+rustc-hash = "1.1.0"
+fast-float = "0.2.0"
+num-traits = "0.2.15"
+bitflags = "1.3.2"
+num-bigint = "0.4.3"
```

```suggestion
num-bigint = "0.4.3"

```
Just a new line at end of file